src/Mapping/Driver/XmlDriver.php

```diff
@@ -914,6 +905,10 @@ protected function loadMappingFile($file)
 
     private function validateMapping(string $file): void
     {
+        if (! $this->isXsdValidationEnabled) {
+            return;
+        }
```

Comment:
Shouldn't this be tested? As far as I understand the proper solution might take some time and we'll have this change for a while.

Reply:
I will check to see how or if it was tested before it was removed and reintroduce the test. If it wasn't tested before then I will write a new test to cover it.

Reply:
Looks like it was not tested before as far as I can tell. I added a test for enabling/disabling xml validation that should take care of it!